drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/actualcommands/AbstractSelectedColumnCommand.java

```diff
@@ -310,4 +314,55 @@ protected String getPropertyHeaderTitle(ScenarioSimulationContext context, List<
                 .findFirst()
                 .map(FactMapping::getExpressionAlias);
     }
+
+    /**
+     * It checks if the provided instance title is valid. A title is valid if not already present in his own column group
+     * (GIVEN/EXPECTED). The only exception is when the selected column is part (i.e on the left or on the right) of
```

Comment:
Verify that in DMN test scenarios each object can be assigned only one time in the whole header.

Reply:
@danielezonca check removed as agreed.